site/classes/attachment.class.php

```diff
@@ -13,7 +13,7 @@
 jimport('joomla.filesystem.folder');
 
 // ensure JemFactory is loaded (because this class is used by modules or plugins too)
-require_once(JPATH_SITE.'/components/com_jem/factory.php');
+require_once(JPATH_SITE . '/components/com_jem/factory.php');
```

Comment:
please use the joomla style guide. So other user and tools (code sniffer ... ) could make better code.